VideoAPI/VideoApi.DAL/Commands/AssignLeaveTimeForRoomParticipantCommand.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Linq;
+using VideoApi.DAL.Commands.Core;
+using VideoApi.DAL.Exceptions;
+using Task = System.Threading.Tasks.Task;
+
+namespace VideoApi.DAL.Commands
+{
+    public class AssignLeaveTimeForRoomParticipantCommand : ICommand
+    {
+        public long RoomId { get; set; }
+        public Guid ParticipantId { get; set; }
+
+        public DateTime LeaveTime { get; set; }
```

Comment:
The leave time should not be provided externally. This should be a method on the consultation room participant.

Reply:
command removed